src/Sylius/Bundle/AdminBundle/Command/CreateAdminUserCommand.php

```diff
@@ -75,46 +71,84 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return Command::INVALID;
     }
 
-    private function createEmailQuestion(): Question
+    private function askAdminUserData(): AdminUserInterface
     {
-        $question = new Question('Email');
-        $question->setValidator(function (?string $email) {
-            if (!filter_var($email, FILTER_VALIDATE_EMAIL) || $email === null) {
-                throw new \InvalidArgumentException('The e-mail address provided is invalid. Please try again.');
-            }
+        /** @var AdminUserInterface $adminUser */
+        $adminUser = $this->adminUserFactory->createNew();
 
-            return $email;
-        });
-        $question->setMaxAttempts(3);
+        $adminUser = $this->askEmail($adminUser);
+        $adminUser = $this->askUsername($adminUser);
+        $adminUser = $this->askFullName($adminUser);
+        $adminUser = $this->askPassword($adminUser);
 
-        return $question;
+        $locales = Locales::getNames();
+        $localeCode = $this->io->choice('Select the locale code', $locales, 'en_US');
```

Comment:
```suggestion
        $localeCode = $this->io->choice('Select the locale code', $locales, $this->defaultLocale);
```
Pass the `%locale%` parameter in the constructor.